package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "boilerplate",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=16.13.2",
+    "pnpm": ">=6.32"
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@infermedica/component-library": "^0.6.9",
+    "axios": "^1.4.0",
+    "vite-svg-loader": "^4.0.0",
+    "vue": "^3.3.4"
```

Comment:
Q: Do we need `vue` if this is going to be _composables-only_ project? And `vite-svg-loader`?  
Or do we even need to build it up with `vite`? 🤔 What do you think?

Reply:
yes; because is a boilerplate not a composable package